libvast/src/detail/make_io_stream.cpp

```diff
@@ -35,15 +43,15 @@ make_input_stream(const std::string& input, bool is_uds) {
                         "failed to connect to UNIX domain socket at", input);
     auto remote_fd = uds.recv_fd(); // Blocks!
     auto sb = std::make_unique<fdinbuf>(remote_fd);
-    return std::make_unique<std::istream>(sb.release());
+    return std::make_unique<owning_istream>(std::move(sb));
   }
   if (input == "-") {
-    auto sb = std::make_unique<fdinbuf>(0); // stdin
-    return std::make_unique<std::istream>(sb.release());
+    auto sb = std::make_unique<fdinbuf>(0);
```

Comment:
Why did you remove the comment? I think it's useful reminder for the reader that FD 0 is standard input.